README

```diff
@@ -1,7 +1,9 @@
 Git Test Project
 Shawn Meier
+Chi Huynh
```

Comment:
I think you should put your middle name in here too.

Reply:
Done.